crates/rooch-framework/sources/nft/nft.move

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+module rooch_framework::nft {
+    use std::string::String;
+    use moveos_std::object_ref;
+    use moveos_std::object;
+    use rooch_framework::collection;
+    use moveos_std::object_ref::ObjectRef;
+    use moveos_std::context::Context;
+    use moveos_std::context;
+    use moveos_std::type_table;
+    use moveos_std::object::ObjectID;
+    use moveos_std::type_table::TypeTable;
+
+    const ENftNotExist: u64 = 100;
+    const EMutatorNotExist: u64 = 101;
+
+    struct NFT has store {
+        name: String,
+        uri: String,
+        collection: ObjectID,
+        creator: address,
+        extend: TypeTable
+    }
+
+    struct MutatorRef has store{
+        nft: ObjectID,
+    }
+
+    struct BurnerRef has store{
```

Comment:
How to get BurnerRef?

Reply:
It's not finished yet, I will add it later.